Tests/test_file_fli.py

```diff
@@ -32,6 +35,24 @@ def test_sanity() -> None:
         assert im.is_animated
 
 
+def test_prefix_chunk() -> None:
+    ImageFile.LOAD_TRUNCATED_IMAGES = True
+    try:
+        with Image.open(animated_flc_with_prefix_chunk_test_file) as im:
```

Comment:
```suggestion
        with Image.open(animated_test_file_with_prefix_chunk) as im:
```

Reply:
Yes, I know, you just changed this - as an explanation, I did edit my previous suggestion a second later, but I'm guessing you just read the email.

Also, you don't have to force push your changes each time. You can apply suggestions like this through the Github interface in the 'Files changed' tab if you want.